cdap-watchdog/src/main/java/io/cdap/cdap/metrics/process/MessagingMetricsProcessorService.java

```diff
@@ -181,7 +180,7 @@ private MetricsConsumerMetaTable getMetaTable() {
 
   @Override
   protected void run() {
-    LOG.info("Start running MessagingMetricsProcessorService");
+    LOG.info("Start running MessagingMetricsProcessorService for " + metricsWriter.getID());
```

Comment:
Use `{}` placeholder for logging.

`LOG.info("Start running MessagingMetricsProcessorService with writer {}", metricsWriter.getID());`

Reply:
Done